### DISCUSSION TITLE
Discord Channel Doesn't work

### DISCUSSION THREAD

**Discussion by johncut06** [zkSync-Community-Hub/zksync-developers#302] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

Discord Channel doesnt work

### What is the specific issue or error you're encountering?

Discord Channel doesnt work

### Can you share the error messages or logs you're receiving, if any?

Discord Channel doesnt work

### Have you made any recent changes to the contract before encountering this issue?

Discord Channel doesnt work

### Are there any external libraries or contracts that your contract interacts with?

Discord Channel doesnt work

### Can you provide the relevant portions of your contract code where the issue is occurring?

Discord Channel doesnt work

### Have you tried to isolate the problem, and if so, what were the results?

Discord Channel doesnt work

### What steps have you already taken to try to resolve the issue?

Discord Channel doesnt work

### Repo Link (Optional)

https://join.zksync.dev/?key=03AFcWeA4X4EhNFK7hF9rpLVSMkMrL0-pg5Itz58OeoL7EUz2KkshrYZmUgEIUCNVC3NB2dN1ffx9EKAlBGxTWMonDtzjPJFVJgRvLMm_bu_cxji4z7DaOdlUM1kROvcDjUHWpSJq8MRM7YyGvzvtWf5CA2i7xOAj21loI9f-CzlEYsuoVMNS5ralK0zCMv1zGKdjdXmOv_03iY5L6bXqxuvZbU_vvT7yVPHI5vl4bBLkNFIigR2vh4cT0-P_wHXUbEVwGUQJZqy-yfV7OV9B64CQT8FbhwESOsJOs9WJWtvpO-c7Qnl6wCTPoVuNJGTR8eLC7KJ9bTGMo_9yyqYeGm1-PsEVTWdZOV-qeDBcKa_hJthJSmZLGrkGNql6zPT-NEBgQfsJ02XKmrfMjGNFBmNt9KX_OJe7NUiDg_tpQmP-O4d8eawzoWZrW1OfPkRVmKAGIkU3Oe5yOKUBrw6aO9ZL68-pLJtrtYyMz-1DNzH9tvDcuppt2OFShKzXCE4CYXM4CM7nY2kQK0LbT38lMwMuGvC5MU8wVmiAWfi0CJVfTV_CXaYwJX99Q-R13wgag4EhpuFCURhMd44r5LpIdPL_gRpbkOwfrmw

**Comment by 0xsisyfos** [comment]
Hello,

Thank you for reaching out to us. 
Make sure you have a profile picture in Discord
Make sure you do not have words such as "admin", "airdrop", etc. in your profile name.
If you were banned, please fill out the form and we'll review as soon as we can https://appeal.zksync.dev/.

Best regards,
